lib/features/copy-paste/ModdleCopy.js

```diff
@@ -234,8 +234,6 @@ ModdleCopy.prototype.copyProperty = function(property, parent, propertyName) {
 
       // copying might NOT be allowed
       if (copiedProperty) {
-        copiedProperty.$parent = parent;
-
```

Comment:
Do we know why we had this in the first place? I think we introduced it via multi-elements copy-paste, but it seems like it's not necessary (/cc @philippfromme). 

It looks good interaction wise and also CI is green.

Reply:
I think it probably was an oversight from how the code is structured but I may be missing something, kinda why I invited flip too. In any case it would be good to find that out, because if there's an issue here we need to write some more tests to cover the cases.

Reply:
I think it probably was an oversight from how the code is structured but I may be missing something, kinda why I invited flip too. In any case it would be good to find that out, because if there's an issue here we need to write some more tests to cover the cases.

Reply:
`$parent` should only be set if `copiedProperty.$type` exists which means it's a moddle element. This issue didn't pop up for a long time because in the standard model all arrays of properties contain only moddle elements (e.g. `extensionElements.values`).